src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -62,38 +66,37 @@ def send_historical_data_to_seer(alert_rule: AlertRule, project: Project) -> Bas
         request_url=SEER_ANOMALY_DETECTION_STORE_DATA_URL,
     )
 
-    snuba_query = SnubaQuery.objects.get(id=alert_rule.snuba_query_id)
-    window_min = int(snuba_query.time_window / 60)
-    historical_data = fetch_historical_data(alert_rule, snuba_query, project)
+    historical_data = fetch_historical_data(snuba_query_data, project_id, organization_id)
 
     if not historical_data:
         base_error_response.reason = "No historical data available. Cannot create alert_rule."
         return base_error_response
 
-    formatted_data = format_historical_data(historical_data)
-
     if (
-        not alert_rule.sensitivity
-        or not alert_rule.seasonality
-        or alert_rule.threshold_type is None
-        or alert_rule.organization is None
+        not rule_data["sensitivity"]
```

Comment:
Is this necessary now that you've built a typeddict?

Reply:
idk i'll ask mypy